terraform/20-app/ecs.service.utility-worker.tf

```diff
@@ -111,54 +111,49 @@ module "ecs_service_utility_worker" {
     }
   ]
 
-  task_exec_iam_statements = {
-    kms_keys = {
+  task_exec_iam_statements = [
+    {
       actions   = ["kms:Decrypt"]
       resources = [
         module.kms_secrets_app_engineer.key_arn,
         module.kms_app_rds.key_arn
       ]
     }
-  }
+  ]
 
-  security_group_rules = {
-    # ingress rules
+  security_group_ingress_rules = {
     alb_ingress = {
-      type                     = "ingress"
       from_port                = 80
       to_port                  = 80
       protocol                 = "tcp"
-      source_security_group_id = module.private_api_alb.security_group_id
+      referenced_security_group_id = module.private_api_alb.security_group_id
     }
-    # egress rules
-    db_egress = {
-      type                     = "egress"
+  }
+  security_group_egress_rules = {
+    db = {
       from_port                = 5432
       to_port                  = 5432
       protocol                 = "tcp"
-      source_security_group_id = module.aurora_db_app.security_group_id
+      referenced_security_group_id = module.aurora_db_app.security_group_id
     }
-    cache_egress = {
-      type                     = "egress"
+    cache = {
       from_port                = 6379
       to_port                  = 6379
       protocol                 = "tcp"
-      source_security_group_id = module.app_elasticache_security_group.security_group_id
+      referenced_security_group_id = module.app_elasticache_security_group.security_group_id
```

Comment:
This attribute is now called `referenced_security_group_id` instead of `source_security_group_id`